recipes/omniscidb-cuda/recipe/meta.yaml

```diff
@@ -0,0 +1,97 @@
+{% set name = "omniscidb-cuda" %}
+{% set version = "4.8.1" %}
+{% set sha256 = "300cdbbaa9108879f7ec22d9cf6210ca2700a3ef99c356a5c00fbf3aeb3a4713" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/omnisci/omniscidb/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    Skip-installing-Boost-libs.patch
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    #- {{ compiler('c') }}
```

Comment:
Why is this commented?

Reply:
IIRC, we want to use `clang` for building omniscidb. There is no excuse for this comment except the PR status is WIP.

Reply:
Why is `clang` preferred?

Reply:
There are several reasons:
1. omniscidb uses clang/llvm for generating machine code at startup. If omniscidb is compiled with gcc, then binary incompatibility between gcc and llvmdev-7 will cause the omniscidb server to crash.
2. On Darwin, clang is the only compiler option. To ease maintenance (of `omniscidb-cpu` that could some day merged with `omniscidb-cuda`), we are using clang on both Linux and Darwin.